codes/csharp/chapter_tree/avl_tree.cs

```diff
@@ -59,28 +59,28 @@ class AVLTree {
     }
 
     /* 执行旋转操作，使该子树重新恢复平衡 */
-    TreeNode? Rotate(TreeNode? node) {
+    static TreeNode? Rotate(TreeNode? node) {
```

Comment:
Remove static for all of the methods in this class, because we don't need to call these methods without constructing an object.

Reply:
fixed